### DISCUSSION TITLE
Animations showing blank image

### DISCUSSION THREAD

**Discussion by LloydOS** [jekhokie/raspberry-noaa-v2#381] [Q&A] [answered]
Since my last clean install of NOAA V2 I am getting blank animations.
Any suggestions?
![animations - blank](https://user-images.githubusercontent.com/80271054/117701455-2aced500-b1bf-11eb-8406-4ba7b34667bb.png)

**Comment by dom-robinson** [comment, marked as answer]
Hey Lloyd - i think you have this working now? Usually you need to wait a few passes until the AVI has 2/3 images processed...
